mr/src/test/java/org/apache/iceberg/mr/TestCatalogs.java

```diff
@@ -212,11 +216,11 @@ public void testLoadCatalogHive() {
         InputFormatConfig.catalogPropertyConfigKey(catalogName, CatalogUtil.ICEBERG_CATALOG_TYPE),
         CatalogUtil.ICEBERG_CATALOG_TYPE_HIVE);
     Optional<Catalog> hiveCatalog = Catalogs.loadCatalog(conf, catalogName);
-    Assert.assertTrue(hiveCatalog.isPresent());
+    Assertions.assertThat(hiveCatalog.isPresent()).isTrue();
```

Comment:
same as above

Reply:
please also update all the other places that are similar to this

Reply:
done